website/content/docs/nia/configuration.mdx

```diff
@@ -197,6 +197,8 @@ A `task` block can be optionally configured with a `condition` block to set the
 
 This condition will trigger the task on services that match the regular expression configured in `regexp` or services listed by name in `names`. Either `regexp` or `names` must be configured, but not both.
 
+When a `condition "services"` is configured for a task, the task cannot be configured with the `services` list field or a `source_input "services"` block. The monitored services information for a task can only be set through one configuration option. Any services source input that the task needs should be configured solely through the `condition` block.
```

Comment:
```suggestion
When a `condition "services"` is applied to a task configuration, then the following restrictions become applicable. The task cannot be configured with the `services` list field or a `source_input "services"` block. The monitored services information for a task can only be set through one configuration option. Any services source input that the task needs should be configured solely through the `condition` block.
```
I tried to make the restrictions more clear.

Reply:
I'm realizing I wasn't clear that `condition "services"` refers to a configuration block. What are your thoughts on adding "block" and keeping "configured for":

> When a `condition "services"` block is configured for a task, then the following restrictions become applicable. The task cannot be configured...

I'll apply whatever we talk about here to the consul-kv suggestion below